packages/material-ui-shell/cra-template-material-ui/template/src/pages/SignIn/SignIn.js

```diff
@@ -135,8 +135,8 @@ const SignIn = () => {
               justifyContent: 'space-between',
             }}
           >
-            <Link to="/password_reset">Forgot Password?</Link>
-            <Link to="/signup">Register</Link>
+            <Link to="/password_reset">{intl.formatMessage({ id: 'forgot_password' })}?</Link>
```

Comment:
You could add here also Default message. It looks nicer if someone uses a different language and forgets to translate some stuff.